region/maridia/outer/Glass Tunnel.json

```diff
@@ -1807,15 +1804,13 @@
           "types": ["ammo"],
           "requires": []
         }
-      ],
-      "note": "Doesn't require canSuitlessMaridia because there is no risk, nor anything tricky whatsoever."
+      ]
     },
     {
       "id": 80,
       "link": [6, 4],
-      "name": "Suitless",
+      "name": "Suitless Spring Ball",
       "requires": [
-        "canSuitlessMaridia",
         "canSpringBallJumpMidAir"
```

Comment:
I thought this strat had trickyJump on it because its not obvious that you have to unmorph.